.github/workflows/build-and-test.yml

```diff
@@ -74,7 +78,7 @@ jobs:
 
       - uses: EndBug/add-and-commit@v4
         with:
-          add: 'docs/*'
+          add: 'dist/apps/docs/*'
```

Comment:
The build output directory is now `dist/apps/docs` instead of the raw files in `docs`.